### PR TITLE
[INLONG-645][Doc] Starrocks load node documents

### LINKED ISSUE #645: [Feature][Doc]Starrocks load node documents
### Description

Documents for starrocks-inlong connector.

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by liaorui**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #645

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by dockerzhang** [comment]
please add Starrocks to the total form.

- introduction
image

- readme
https://github.com/apache/inlong
image

File: docs/introduction.md [hunk 1]
```diff
@@ -91,4 +91,6 @@ Apache InLong serves the entire life cycle from data collection to landing,  and
 |              | Greenplum         | 4.x, 5.x, 6.x                | Lightweight, Standard |
 |              | Elasticsearch     | 6.x, 7.x                     | Lightweight, Standard |
 |              | SQLServer         | 2012, 2014, 2016, 2017, 2019 | Lightweight, Standard |
+|              | Doris             | >= 0.13                      | Lightweight, Standard |
+|              | StarRocks         | >= 2.0                       | Lightweight, Standard |
```
Comment (dockerzhang) [inline docs/introduction.md:95]: please update the related Chinese document at the same time.

https://github.com/apache/inlong-website/blob/master/i18n/zh-CN/docusaurus-plugin-content-docs/current/introduction.md

File: docs/data_node/load_node/starrocks.md [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+
+### Maven dependency
+
+```
+<dependency>
+    <groupId>org.apache.inlong</groupId>
+    <artifactId>sort-connector-starrocks</artifactId>
+    <version>${siteVariables.inLongVersion}</version>
+</dependency>
+```
```
Comment (dockerzhang) [inline docs/data_node/load_node/starrocks.md:34]: <pre><code parentName="pre">
{`<dependency>
    <groupId>org.apache.inlong</groupId>
    <artifactId>sort-connector-starrocks</artifactId>
    <version>${siteVariables.inLongVersion}</version>
</dependency>
`}
</code></pre>

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/starrocks.md [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+
+### Maven 依赖
+
+```
+<dependency>
+    <groupId>org.apache.inlong</groupId>
+    <artifactId>sort-connector-starrocks</artifactId>
+    <version>${siteVariables.inLongVersion}</version>
+</dependency>
+```
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/starrocks.md:32]: ditto

**Review by dockerzhang — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

**Pull request merged**